test/enhance/test_adjust.py

```diff
@@ -742,3 +742,86 @@ def build_input(channels, depth, height, width, bs=1, row=None):
         batch = torch.stack([image3d] * bs)
 
         return batch
+
+
+class TestSharpness(object):
+
+    f = kornia.enhance.sharpness
+
+    @pytest.mark.parametrize("channels", [1, 3, 5])
+    def test_shape(self, channels, device, dtype):
```

Comment:
I guess you can merge this two tests into one. Would be great to inherit from BaseTester and have same test names. Recently learnt that we can filter by test names which will make easier later for the ci. See: https://github.com/kornia/kornia/pull/787/files#diff-fd3c5292dc69298c69ca985ded96d320072ac2625604af20c5bbbef1cdaff562R26